<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDescriptionWindow</class>
 <widget class="QDialog" name="EventDescriptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BuyTicketButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy ticket</string>
   </property>
  </widget>
  <widget class="QLabel" name="EventName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Event name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tickets">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tickets</string>
   </property>
  </widget>
  <widget class="QLabel" name="isFinishedStatus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Event status</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelForEventName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelForTicketsCount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelForStatus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TicketPrice">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelForTicketPrice">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
